app/meteor-accounts-saml/server/definition/IServiceProviderOptions.ts

```diff
@@ -9,18 +9,16 @@ export interface IServiceProviderOptions {
 	customAuthnContext: string;
 	authnContextComparison: string;
 	defaultUserRole: string;
-	roleAttributeName: string;
-	roleAttributeSync: boolean;
-	allowedClockDrift: number;
-	signatureValidationType: string;
-	identifierFormat: string;
-	nameIDPolicyTemplate: string;
-	authnContextTemplate: string;
-	authRequestTemplate: string;
-	logoutResponseTemplate: string;
-	logoutRequestTemplate: string;
-	metadataCertificateTemplate: string;
-	metadataTemplate: string;
+	allowedClockDrift?: number;
+	signatureValidationType?: string;
+	identifierFormat?: string;
+	nameIDPolicyTemplate?: string;
```

Comment:
keep the template options as mandatory, just set them to their default values when in CE.